### ISSUE TITLE
HIMS SmartBeetle Fails To Load In 2017.3

### ISSUE THREAD

**Issue by munawarb** [nvaccess/nvda#7663]
### Steps to reproduce:
- Connect SmartBeetle via USB and switch SmartBeetle to USB Mode.
- Select HIMS BrailleSense/.../SmartBeetle from Braille display options and click "OK." NVDA gives an error message.
- Install the HIMS Braille Display Driver and select the SmartBeetle again, error reappears

### Error Log
ERROR - braille.BrailleHandler.setDisplayByName (18:37:36.155):
Error initializing display driver
Traceback (most recent call last):
  File "braille.pyo", line 1481, in setDisplayByName
  File "brailleDisplayDrivers\hims\__init__.pyo", line 140, in __init__
RuntimeError: No display found
INFO - braille.BrailleHandler.setDisplayByName (18:37:36.155):
Loaded braille display driver noBraille, current display has 0 cells.

### System configuration:
NVDA version: 2017.3

NVDA Installed or portable: Installed

Windows version: Windows 10 Creator's Update

### Other questions:

Does the issue still occur after restarting your PC? Yes

Have you tried any other versions of NVDA? No

**Comment by derekriemer** [comment]
cc @dkager

**Comment by leonardder** [comment]
Could you please test this with NVDA 2017.2 and with debug logging enabled? Nothing has changed to the Hims driver that should have caused such an issue.

**Comment by munawarb** [comment]
@leonardder Ok. I'm heading toward the possibility that it's something on my end. I have a sync cable coming today (I think the one I'm using is charge-only,) and also a bluetooth adapter. I will try both and let you know if it fixes it.

**Comment by munawarb** [comment]
I tried the Bluetooth adapter and it connected with NVDA just fine. So the issue must have been the cable. Closing the issue.
